src/main/java/com/paskar/email/application/service/EmailService.java

```diff
@@ -0,0 +1,46 @@
+package com.paskar.email.application.service;
+
+
+import com.paskar.email.application.model.Email;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class EmailService {
+    private static final Logger LOG = LoggerFactory.getLogger(EmailService.class);
+
+    private final JavaMailSender emailSender;
+    private final EmailRepositoryJdbcImpl emailDao;
```

Comment:
Please avoid `dao`. Let us not mix patterns. You should use repository pattern, not DAO.